### DISCUSSION TITLE
How to make Cmus automatically add new tracks?

### DISCUSSION THREAD

**Discussion by Serif-7** [cmus/cmus#1084] [General]
I occasionally add new tracks from Youtube via `youtube-dl`. Every time I do this I have to do ':add (folder)' which is annoying. Is there an 'rc' file for cmus that I can use, like for Vim?

**Comment by pgaskin** [comment]
cmus does have a RC file in the config dir. Adding the `add` command to it (without the colon) should theoretically work.
